src/btreemap.rs

```diff
@@ -2388,6 +2444,18 @@ mod test {
         }
     }
 
+    #[test]
+    fn test_iter_upper_bound() {
+        let mut stable_map = super::BTreeMap::new(make_memory());
+        for k in 0..1000u64 {
+            stable_map.insert(k, ());
+            println!("Getting a upper bound for {}", k + 1);
```

Comment:
Debug leftover?

Reply:
These prints are helpful for the case when the test fails. Turned those into better assert messages.